<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageAugment</class>
 <widget class="QWidget" name="ImageAugment">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Augmentor</string>
  </property>
  <widget class="QCheckBox" name="hflipCb">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>92</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Waree</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>hflip</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="vflipCb">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>92</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Waree</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>vflip</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="renameCb">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>92</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Waree</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>rename</string>
   </property>
  </widget>
  <widget class="QPushButton" name="runBtn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Waree</family>
    </font>
   </property>
   <property name="text">
    <string>RUN</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>371</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="pathLayout">
    <item>
     <widget class="QLabel" name="pathLable">
      <property name="font">
       <font>
        <family>Waree</family>
       </font>
      </property>
      <property name="text">
       <string>PATH</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="pathLine"/>
    </item>
    <item>
     <widget class="QPushButton" name="pathBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>â˜ž</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>361</width>
     <height>60</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="renameLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="prefixLabel">
      <property name="font">
       <font>
        <family>Waree</family>
       </font>
      </property>
      <property name="text">
       <string>prefix</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="suffixLabel">
      <property name="font">
       <font>
        <family>Waree</family>
       </font>
      </property>
      <property name="text">
       <string>suffix</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="prefixLine">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="numberLabel">
      <property name="font">
       <font>
        <family>Waree</family>
       </font>
      </property>
      <property name="text">
       <string>number</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="suffixLine">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>231</width>
     <height>35</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QCheckBox" name="resizeCb">
      <property name="minimumSize">
       <size>
        <width>77</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>77</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Waree</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Resize</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="wresizeLine">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>211</red>
            <green>215</green>
            <blue>207</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>211</red>
            <green>215</green>
            <blue>207</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>211</red>
            <green>215</green>
            <blue>207</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>190</red>
            <green>190</green>
            <blue>190</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string>width</string>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="hresizeLine">
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string>height</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>231</width>
     <height>35</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QCheckBox" name="rotateCb">
      <property name="minimumSize">
       <size>
        <width>74</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>74</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Waree</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>rotate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="rotateLine"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
